.github/workflows/test.yml

```
@@ -19,7 +19,7 @@ jobs:
     strategy:
       matrix:
         platform: ["ubuntu-latest", "windows-latest"]
-        python-version: [ "3.6", "3.7", "3.8", "3.9", "3.10" ]
+        python-version: [ "3.6", "3.7", "3.8", "3.9", "3.10", "3.11" ]
```

```suggestion
        python-version: [ "3.6", "3.7", "3.8", "3.9", "3.10", "3.11-dev" ]
```

Looks like we need this until 3.11 is actually released. I'll commit this change myself to test it quickly.